images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1

```diff
@@ -21,4 +21,26 @@ function Get-GeckodriverVersion {
 function Get-ChromiumVersion {
     $chromiumVersion = chromium --version | Take-OutputPart -Part 0,1
     return $chromiumVersion
-}
\ No newline at end of file
+}
+
+function Build-BrowserWebdriversEnvironmentTable {
+    return @(
+        @{
+            "Name" = "CHROMEWEBDRIVER"
+            "Value" = $env:CHROMEWEBDRIVER
+        },
+        @{
+            "Name" = "EDGEWEBDRIVER"
+            "Value" = $env:EDGEWEBDRIVER
+        },
+        @{
+            "Name" = "GECKOWEBDRIVER"
+            "Value" = $env:GECKOWEBDRIVER
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```

Comment:
These variables are not symlinks so we don't need to use `Get-PathWithLink` here